pwa/src/templates/categoriesDetailTemplate/CategoriesDetailTemplate.module.css

```diff
@@ -26,12 +26,24 @@
   margin-block-start: var(--web-app-size-xl);
 }
 
+.solutions > *:not(:last-child) {
+  margin-block-end: var(--web-app-size-xl);
+}
+
+.viewButtons {
+  display: flex;
+  justify-content: end;
+  gap: var(--web-app-size-md);
```

Comment:
The gap can have the size of xs `--web-app-size-xs` to keep it inline with the other buttons.